posts/on-limit-collaboration/index.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Collaboration with the ON-LiMiT project :tada:"
+description: |
+    The Novo Nordisk Foundation has given researchers across different research
+    centers, such as Steno Aarhus, a grant to run a large study to see how
+    lifestyle changes can help with diabetes remission and management.  Like
+    with DP-Next, the Seedcase eam will be collaborating with them on their
+    data infrastructure, engineering, and architectural needs.
+author:
+- Luke W. Johnston
+date: "2025-05-28"
+categories:
+  - collaborations
+  - external projects
+---
+
+Novo Nordisk Foundation has given a
+[grant](https://novonordiskfonden.dk/en/news/new-study-seeks-answers-on-the-role-of-diet-and-exercise-in-diabetes-remission/)
+to several centers, including [Steno Diabetes Center
+Aarhus](https://www.stenoaarhus.dk/), to conduct a large study on how
+lifestyle changes can help people with diabetes. This is the summary of
+the project:
+
+> "ON LiMiT (Optimal Non-pharmacological LifeStyle Modifications in
+> people with Type 2 diabetes), is a large randomized controlled
+> intervention study, conducted in collaboration with the Steno Diabetes
+> Centre in Aarhus, Odense and Copenhagen, as well as the University of
+> Copenhagen and Bispebjerg Hospital.
+>
+> The trial aims to investigate whether it is possible to achieve and
+> maintain remission of type 2 diabetes through weight loss and
+> lifestyle changes." --- [ON-LiMiT](https://onlimit.dk) (translated
+> from Danish)
```

Comment:
Wasn't there a problem with the url, and they had to change away from a .dk domain?